rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -116,17 +116,21 @@ error_code sys_rsx_context_allocate(vm::ptr<u32> context_id, vm::ptr<u64> lpar_d
 	std::memset(&reports, 0, sizeof(RsxReports));
 
 	for (int i = 0; i < 64; ++i)
-		reports.notify[i].timestamp = (u64)-1;
+		reports.notify[i].timestamp = -1;
```

Comment:
Now -1 isn't properly sized, so you have to add ull

Reply:
A better alternative here is ~0ull or uint64_max

Reply:
There's no undefined behaviour here, solved in discord.